jest.config.js

```diff
@@ -0,0 +1,185 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Do we need all these comments? It seems more clear to me to only include used config options

Reply:
Agreed!